NEMO/templates/tool_control/tool_control.html

```diff
@@ -20,8 +20,9 @@
 	{% else %}
 		<div id="sidebar" class="application-sidebar">
 			<div id="expand-collapse" class="btn-group sidebar-item">
```

Comment:
the button is shown at all times, it should only be shown when the option is checked in customizations

Reply:
Filtering is now applied to the tool control as well